ui/app/services/token.js

```diff
@@ -4,20 +4,6 @@ import fetch from 'fetch';
 const { Service, computed, assign } = Ember;
 
 export default Service.extend({
-  accessor: computed({
```

Comment:
so with the move from accessor -> token, you'll just always do the lookup at the server now?

Reply:
The lookup was always done on the server. Just now the accessor isn't necessary, due to a new `/acl/token/self` endpoint that looks for a token with a matching secret.